modules/Hoot/ui/managePanel/datasets.js

```diff
@@ -257,7 +257,7 @@ export default class Datasets extends Tab {
             }
             case 'importDatasets': {
                 let translations = await Hoot.api.getTranslations();
-                this.importMultiModal = new ImportMultiDataset( translations, d.data.name ).render();
+                this.importMultiModal = new ImportMultiDataset( translations, d.data ).render();
```

Comment:
why not just pass `d.data.path` here instead of `d.data.name`?
I don't see the advantage of passing the entire object.